.github/stale.yml

```diff
@@ -0,0 +1,9 @@
+onlyLabels: ['Status: Waiting feedback']
+daysUntilStale: 14
+daysUntilClose: 7
+staleLabel: 'Won''t Fix'
+markComment: >
+  We requested feedback a while ago and as of now it's still not clear to us what needs to be done.
```

Comment:
Suggested rewording:

> This issue is still waiting on feedback. Please provide the information requested above to allow us to continue working on this ticket.

This issue will be closed after 7 days. You will still be able to reopen the issue and provide us with the information needed to work on this issue.

Reply:
I don't think they will be able to reopen the issue. I will rather leave out that part.

Reply:
They will be, and they should if they want to follow up with the information they requested.

Reply:
IIRC they can open new issue, not reopen their issue when somebody else closes it

Reply:
They can reopen their issue just fine. Issue has to be locked to prevent reopening, which we generally don’t do. With history already in the ticket, I’d advise against requesting OP open a new ticket for the same issue but rather reopen if new information comes up.

Reply:
IIRC, when an issue is closed by a collaborator, only collaborators can reopen them, not the issue author. They can reopen only what they closed themselves (unless Github changed that in the past few months, as I haven't tried it again recently).

Reply:
This. Users can not really reopen issue when somebody else closes it, that's misconception. Plenty times my issue was closed and I had to open new issue.

Reply:
Welp, I was sure this would work.

> This issue is still waiting on feedback. Please provide the information requested above to allow us to continue working on this ticket.
>
> This issue will be closed after 7 days. You can still provide us with the necessary information to continue working on this issue, and we will reopen the ticket for you. Thanks!